packages/simcore-sdk/tests/node_ports/docker-compose.yml

```diff
@@ -1,7 +1,12 @@
 version: '3.6'
 services:
+  #TODO: use same integration test framework as in web server instead of fixed docker-compose file
```

Comment:
I would add the defaults in all ``docker_compose`` files **used as pytest fixtures** as
```bash
${DOCKER_REGISTRY:-itisfoundation}
${DOCKER_IMAGE_TAG:-latest}
```

Two reason:
 a) you avoid writing ``image_environ(...)`` fixture in conftest
 b) sometimes while developing I deploy these services by hand to speed up the test process. in that case, current solution would not work

Reply:
well with this stuff I am not yet satisfied. Basically I would like:
1. that the node ports integration tests do use the same system as the other integration tests instead of using a fixed docker-compose file
2. it would be amazing if the tests could actually use the makefile instead of directly using the docker-compose files... 

But I will do what you ask here, that should fix your issue. agreed.

Reply:
done